planning/behavior_velocity_crosswalk_module/src/scene_crosswalk.cpp

```diff
@@ -358,6 +366,14 @@ std::optional<StopFactor> CrosswalkModule::checkStopForCrosswalkUsers(
       if (collision_state != CollisionState::YIELD) {
         continue;
       }
+      if (ego_crosswalk_passage_direction) {
+        const double direction_diff = tier4_autoware_utils::normalizeRadian(
```

Comment:
It requires abs().

In addition, depends on the significance of this PR, mod() by \pi rad may be required.